src/Maestro/Maestro.MergePolicies/StandardMergePolicy.cs

```diff
@@ -2,55 +2,73 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using Maestro.Contracts;
+using Newtonsoft.Json.Linq;
 using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
 
 namespace Maestro.MergePolicies
 {
-    public class StandardMergePolicy : MergePolicy
+    public class StandardMergePolicyBuilder : IMergePolicyBuilder
     {
-        public StandardMergePolicy()
+        private static readonly MergePolicyProperties s_standardGitHubProperties;
+        private static readonly MergePolicyProperties s_standardAzureDevOpsProperties;
+
+        public string Name => MergePolicyConstants.StandardMergePolicyName;
+
+        static StandardMergePolicyBuilder()
         {
-            standardGithubIgnoredChecks = new HashSet<string>
+            s_standardGitHubProperties = new MergePolicyProperties(new Dictionary<string, JToken>
             {
-                "WIP",
-                "license/cla",
-                "auto-merge.config.enforce"
-            };
-            standardAzureDevOpsIgnoredChecks = new HashSet<string>
+                { 
+                    "ignoreChecks", 
+                    JToken.FromObject(new []
```

Comment:
I think this can be `new JArray("WIP", "license/cla", "auto-merge.config.enfoce")`, which avoids the new anonymous type and the need to serialize it.

Reply:
new [] {..} it is not anonymous type but shortcut for new string[] {...}, but you are right new JArray is better here